BCnEncTests/DecodingAsyncTests.cs

```diff
@@ -0,0 +1,76 @@
+using System.IO;
+using BCnEncoder.Decoder;
+using BCnEncoder.Encoder;
+using BCnEncoder.Shared;
+using SixLabors.ImageSharp;
+using Xunit;
+
+namespace BCnEncTests
+{
+	public class DecodingAsyncTests
+	{
+		[Fact]
+		public async void DecodeAsync()
+		{
+			var decoder = new BcDecoder();
+			var encoder = new BcEncoder();
+			var original = ImageLoader.TestGradient1;
+
+			var file = encoder.EncodeToKtx(original);
+			var image = await decoder.DecodeAsync(file);
+
+			var psnr = TestHelper.CalculatePSNR(original, image);
+			if (encoder.OutputOptions.Quality == CompressionQuality.Fast)
```

Comment:
Should this check be extracted to a helper method? Seems like it's everywhere.

Reply:
I first wanted to get the tests running successfully, before I refactor further.

Reply:
Extracted to common methods in newest commit now.